starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,84 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+.foLogo{
+  background-image: url(../images/soup.jpg);
```

Comment:
background image was a bit small. A good width for the div would be about 72vw - 75vw.